app/views/markets/edit.html.erb

```diff
@@ -0,0 +1,35 @@
+<main>
+	<div class="tittle"><h2>Edit <%= @market.name %> </h2></div>
+	<div class="form">
+	<%= form_for @market do |f| %>
+
+	<%= f.label :name %>
+	<%= f.text_field :name %>
+	<br>
```

Comment:
Try not to use `<br />` because it conceals styling on the page within the view. For clarity it's good to keep all your styling in one place, the .css files, because if you return to this project later you'll know just where everything is and won't have to search for it! For the same reason we shy away from inline CSS styling.